### PR TITLE
Add destinationDir to saltpack frontend file RPCs

### PR DESCRIPTION
**Pull request by patrickxb**
cc @thebearjew

### CODE REVIEW

File: protocol/avdl/keybase1/saltpack.avdl [hunk 1]
```diff
@@ -96,7 +96,7 @@ protocol saltpack {
 
   SaltpackEncryptFileResult saltpackEncryptStringToTextFile(int sessionID, string plaintext, SaltpackFrontendEncryptOptions opts);
 
-  SaltpackEncryptFileResult saltpackEncryptFile(int sessionID, string filename, SaltpackFrontendEncryptOptions opts);
+  SaltpackEncryptFileResult saltpackEncryptFile(int sessionID, string filename, string destinationDir, SaltpackFrontendEncryptOptions opts);
```
Comment (mlsteele) [inline protocol/avdl/keybase1/saltpack.avdl]: // destinationDir (optional) If unset then the result will be put in the same directory as `filename`.

Comment (patrickxb) [reply protocol/avdl/keybase1/saltpack.avdl]: i think you and I are the only ones that read avdl for the interface spec, but sure.

Comment (thebearjew) [reply protocol/avdl/keybase1/saltpack.avdl]: Hey I do it too!

**Comment by patrickxb** [comment]
I don't know what the CI error is, so i'm just going to merge.

**Pull request merged**